config/application.rb

```diff
@@ -0,0 +1,35 @@
+require_relative 'boot'
+
+require "rails"
+# Pick the frameworks you want:
+require "active_model/railtie"
+require "active_job/railtie"
+require "active_record/railtie"
+require "active_storage/engine"
+require "action_controller/railtie"
+require "action_mailer/railtie"
+require "action_view/railtie"
+require "action_cable/engine"
+# require "sprockets/railtie"
+# require "rails/test_unit/railtie"
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module GnosisApi
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.load_defaults 5.2
+
+    # Settings in config/environments/* take precedence over those specified here.
```

Comment:
You should add configuration that the application does not generate unnecessary tests.

Reply:
Please, review. Have modified the file

Reply:
<img width="429" alt="Screenshot 2019-08-13 at 14 59 36" src="https://user-images.githubusercontent.com/50490216/62943633-39fed400-bddb-11e9-88c9-8ae6baf61c94.png">
Source: [bundler official](https://bundler.io/blog/2018/11/04/an-update-on-bundler-2.html)